app/scripts/lib/manifestFlags.ts

```diff
@@ -0,0 +1,20 @@
+export type ManifestFlags = {
+  circleci?: {
+    enabled: boolean;
+    branch?: string;
+    buildNum?: number;
+    job?: string;
+    nodeIndex?: number;
+    prNumber?: number;
+  };
+  doNotForceSentryForThisTest?: boolean;
+};
+
+/**
+ * Get the runtime flags that were placed in manifest.json by alterBuiltManifest.ts
+ *
+ * @returns flags if they exist, otherwise an empty object
+ */
+export function getManifestFlags(): ManifestFlags {
+  return chrome.runtime.getManifest()._flags || {};
```

Comment:
Instead of adding `chrome` to the `scuttleGlobalThisException`, which may require involving the lavamoat team in the review, I think you can use the `browser` polyfill:

```
import browser from 'webextension-polyfill';

... browser.runtime.getManifest() ...
```

Reply:
Hm. Was the type error because of `_flags`? If so we can extend the `browser.Manifest.WebExtensionManifest` type with the `_flags` property, or `// @ts-nocheck` it.

Reply:
Ooh nice I figured it out with that hint